server/tables/tables.go

```diff
@@ -243,6 +243,24 @@ func (t *Token) TableName() string {
 	return "Tokens"
 }
 
+type APIKey struct {
+	Model
+
+	APIKeyID string `gorm:"primary_key"`
+	UserID   string
+	GroupID  string `gorm:"index:api_key_group_id_index"`
+	Perms    int
+	// The API key token used for authentication.
+	Value string `gorm:"unique_index:api_key_value_index"`
```

Comment:
Is there a reason to separate the Value from the APIKeyID since they are both strings and both the same value?

Reply:
Will double check the code that I wrote but I think `APIKeyID` and `Value` should be different - the ID follows the standard `PrimaryKeyForTable` convention (`AK12345...`) and the `Value` is the 20-char alphanumeric token that we generate, and the one that's actually used for auth